lang/en/bigbluebuttonbn.php

```diff
@@ -31,6 +31,8 @@
 $string['bbbrecordwarning'] = 'This session may be recorded.';
 $string['bigbluebuttonbn:addinstance'] = 'Add a new bigbluebuttonbn room/activity';
 $string['bigbluebuttonbn:join'] = 'Join a bigbluebuttonbn meeting';
+$string['bigbluebuttonbn:room'] = 'Create instances as a Room/Activity only';
+$string['bigbluebuttonbn:record'] = 'Create instances as a Recordings only';
```

Comment:
And here:
$string['bigbluebuttonbn:room'] = 'Allow user to create instances with Recording capabilities';